bin/jenkins/cloud-redeploy.sh

```diff
@@ -14,6 +17,8 @@ docker-compose -f ./config/jenkins/cluster.yml down
 
 docker-compose -f ./config/jenkins/cluster.yml up -d
 
+docker-compose -f ./config/jenkins/cluster.yml scale jenkins-slave-xenial=5
```

Comment:
I guess we can do so when we switch to the ephemeral slaves, they can be more lightweight and specialized, and in that case only having one executor makes sense. Right now it would mean having x2 slaves (and jvm instances) running to have the same amount of executors as we have now.